src/apis/auth.ts

```diff
@@ -1,30 +1,29 @@
+import { UserProps } from '../hooks/useAuth';
 import { LoginProps } from '../pages/User/Login';
-import { SignupProps } from '../pages/User/Signup';
 import { httpClient } from './http';
 
-export const join = async (userData: SignupProps) => {
+export const join = async (userData: UserProps) => {
   try {
-    const response = await httpClient.post(`/users/join`, userData);
-    console.log(httpClient);
+    const headers = {
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${userData.uid}`,
+    };
+    const body = {
+      nickname: userData.nickname,
+      image: userData.image,
+      team: userData.team,
+    };
+    const response = await httpClient.post(`/users/join`, body, {
+      headers,
+    });
```

Comment:
여기서는 header와 body를 변수에 담는게 큰 의미가 있다고 느껴지지는 않네요!
저는 post 메소드에 그대로 넣어줘도 괜찮을 것 같아요~
(코드 포맷팅은 무시해주세요!)
```suggestion
    const response = await httpClient.post(`/users/join`, 
    {
      nickname, 
      image, 
      team
     }, 
    {
      headers : {
        'Content-Type': 'application/json',
        Authorization: `Bearer ${userData.uid}`
      }
    });
```